.github/workflows/build.yml

```diff
@@ -39,3 +39,26 @@ jobs:
         - name: Test
           shell: bash
           run: cd build && ctest --output-on-failure
+
+  bz-build:
+      strategy:
+        matrix:
+          os: [ubuntu-latest, windows-latest, macos-latest]
+      runs-on: ${{ matrix.os }}
+      steps:
+        - uses: actions/checkout@v2
+
+        - name: Build
+          shell: bash
+          run: |
+            cd "${{ github.workspace }}"
+            bazel build '//:yaml-cpp'
```

Comment:
`bazel build ...`

is more common, I believe.

Reply:
Here, I only got `bazel build :all` to work as expected.